geode-core/src/main/java/org/apache/geode/cache/configuration/StringType.java

```diff
@@ -57,6 +57,12 @@
   @XmlValue
   protected String value;
 
+  public StringType() {}
+
+  public StringType(String value) {
```

Comment:
I personally dislike the disparity that some configuration objects are deemed deserving of parameterized constructors and others are not.  Given that the rest of the configuration objects are all build declaratively, I think we should be consistent with that.

Alternatively, we should introduce parameterized constructors to all configuration objects, rather than adding them piecemeal as we decide we want them.

Reply:
iteratively, yes. We don't have to create all those parameterized constructors all at once. Create when needed.

Reply:
Also, maybe we can git rid of this type altogether. After all, what is it different than just String? but maybe create another ticket to address it.

Reply:
As long as we're trying to respect the existing cache XSD in our underlying implementation, I think we need this class.  Otherwise the marshaled XML will have a `xsd:string` element rather than `gf:string-type`.

But whenever we break away, then yeah, straight `String` al the way.